README.rst

```diff
@@ -228,16 +230,31 @@ configured as above, but with the following code::
     {# Custom CSS overrides #}
     {% set css_files = css_files + ['_static/my-styles.css'] %}
 
-**Sphinx >= 1.6**
+.. warning::
 
-Add a `setup` function in "conf.py" with stylesheet paths added relative to the
+   The Sphinx Bootstrap Theme does not provide legacy Sphinx support, the
```

Comment:
Is this correct?

My understanding is that the new `@import` CSS sheets put into place after #159 works for both sphinx <= 1.5 and sphinx > 1.5. It's just the "how do I override" instructions that differ...